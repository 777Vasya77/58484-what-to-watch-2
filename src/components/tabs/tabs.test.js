import React from 'react';
import renderer from 'react-test-renderer';
import Tabs from '~/components/tabs/tabs';

describe(`Tabs component tests`, () => {
  it(`Component render correctly`, () => {
    const tree = renderer
        .create(
            <Tabs activeTabName={`Details`} filmId={1}/>
        ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
