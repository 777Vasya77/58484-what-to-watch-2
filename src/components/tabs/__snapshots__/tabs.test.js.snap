// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs component tests Component render correctly 1`] = `
<ul
  className="movie-nav__list"
>
  <li
    className="movie-nav__item false"
  >
    <a
      className="movie-nav__link"
      data-tab-name="overview"
      href="#overview"
      onClick={[Function]}
    >
      Overview
    </a>
  </li>
  <li
    className="movie-nav__item false"
  >
    <a
      className="movie-nav__link"
      data-tab-name="details"
      href="#details"
      onClick={[Function]}
    >
      Details
    </a>
  </li>
  <li
    className="movie-nav__item false"
  >
    <a
      className="movie-nav__link"
      data-tab-name="reviews"
      href="#reviews"
      onClick={[Function]}
    >
      Reviews
    </a>
  </li>
</ul>
`;
